### ISSUE TITLE
Integrate CNCF blog article as comprehensive kepler overview

### ISSUE THREAD

**Issue by BGrasnick** [sustainable-computing-io/kepler-doc#112]
Hey, 

after reading [this CNCF blog article](https://www.cncf.io/blog/2023/10/11/exploring-keplers-potentials-unveiling-cloud-application-power-consumption/), I think it would make for a great starting point when people check out kepler. 

We could then from there link to more specific design subpages, what do you think?

**Comment by sunya-ch** [comment]
Agree. I think it worth putting the link to it at the top page too (https://sustainable-computing.io).

**Comment by BGrasnick** [comment]
Good idea! Putting a link there is a good first move. Later on, we can integrate the content into the site with crosslinks to more details.

**Comment by iacker** [comment]
This one looks done. @sunya-ch's suggestion of putting the link on the top page landed — `docs/index.md:9` currently reads:

> For a comprehensive overview, please check out ➡️ [this CNCF blog article](https://www.cncf.io/blog/2023/10/11/exploring-keplers-potentials-unveiling-cloud-application-power-consumption/).

It's live on https://sustainable-computing.io and the article still resolves (200). The piece is also listed in `docs/project/resources.md` alongside the other blog posts and papers.

The deeper follow-up from the thread — folding the article's content into the site with crosslinks to the design pages — hasn't happened, so worth keeping open if that's still the goal. If the intent was just "link it prominently", that's covered.
